app/representers/api/v1/find_or_create_user_representer.rb

```diff
@@ -12,30 +12,36 @@ class FindOrCreateUserRepresenter < Roar::Decorator
              readable: true,
              writeable: false
 
-    property :email,
+    property :external_id,
              type: String,
              readable: false,
              writeable: true,
              schema_info: {
-               required: true,
-               description: "Email address to search by or assign to newly created user"
+               description: "External ID to search by or assign to newly created user"
              }
 
-    property :already_verified,
-             type: :boolean,
+    property :username,
```

Comment:
do we / should we continue to have usernames in the API? Is someone still using these?

Reply:
Tutor uses them when creating preview course students